src/daemons/StorageDaemon.cpp

```diff
@@ -62,6 +66,14 @@ int main(int argc, char *argv[]) {
     return EXIT_FAILURE;
   }
 
+#if defined(__x86_64__)
+  status = setupBreakpad();
```

Comment:
Do we need this in Debug mode?

Reply:
It's useful too, because gdb is not always available.